meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -1,4 +1,12 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:${THISDIR}/files:"
+
+SRC_URI_append_imx = " \
+    file://0001-optee-obtain-emmc-rpmb-info-from-dt.patch \
+"
```

Comment:
not sure about this append

Reply:
It would apply to all of our `imx` targets which I think is fine as we set PREFERRED_VERSION for `u-boot-fio` for all `imx` with: https://github.com/foundriesio/meta-lmp/blob/master/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc#L399